# Data path. adjust them!
common:
    # image_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/lidar_cam_0118_1/5000.png"
    # pcd_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/lidar_cam_0118_1/0.pcd"
    # result_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/lidar_cam_0118_1/extrinsic_V2.txt"

    # image_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/lidar_cam_0118_2/2200.png"
    # pcd_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/lidar_cam_0118_2/0.pcd"
    # result_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/lidar_cam_0118_2/extrinsic_V2.txt"

    # image_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/indoor_1/1.png"
    # pcd_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/indoor_1/3.pcd"
    # result_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/indoor_1/extrinsic_V2.txt"

    # image_file:  "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/indoor_2/2.png"
    # pcd_file:    "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/indoor_2/0.pcd"
    # result_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/indoor_2/extrinsic_V2.txt"


    image_file:  "/home/bag/livox_hk_sync_test/cab/0.jpg"
    pcd_file:    "/home/bag/livox_hk_sync_test/cab/0.pcd"
    result_file: "/home/bag/livox_hk_sync_test/cab/extrinsic_v2_1.txt"

    # image_file:  "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/denser_pic_3/0.png"
    # pcd_file:    "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/denser_pic_3/001.pcd"
    # result_file: "/home/liunao/calib/src/livox_camera_calib/result/cabli_V2/denser_pic_3/extrinsic.txt"

# Camera Parameters. Adjust them!
# v1
# camera:
#     camera_matrix: [864, 0.0,  351,
#                     0.0, 865,  256,
#                     0.0, 0.0,  1.0 ]
#     dist_coeffs: [0.3175,   -2.77,  7.22e-03, 4.136e-03,  3.1]

# v2
# camera:
#     camera_matrix: [430.885, 0.0,      324.299,
#                     0.0,     431.067,  252.805,
#                     0.0,     0.0,      1.0]
#     dist_coeffs: [-0.058, 0.1031, -0.00066825, -0.04018816, -0.000887]
# v2--liejian
# camera:
#     camera_matrix: [596.232535, 0.0,      376.027289,
#                     0.0,     598.249756,  277.115153,
#                     0.0,     0.0,      1.0]
#     dist_coeffs: [-0.088744 , 0.0626990 , -0.002823 ,  0.0012220 , 0]

# v2_2nd  vanjee+hk
camera:
    camera_matrix: [620.493585,   0.    , 314.065225,
                    0.     , 621.346845 , 256.85171,
                    0.     ,   0.      ,   1     ]
    dist_coeffs: [-0.097161,0.148119,-0.0017345,0.000604,0]

# Calibration Parameters.!
calib:
    # calib_config_file: "/home/liunao/calib/src/livox_camera_calib/config/config_indoor.yaml"
    calib_config_file: "/home/liunao/calib/src/livox_camera_calib/config/config_outdoor.yaml"
    use_rough_calib: true # set true if your initial_extrinsic is bad
